corehq/apps/app_manager/util.py

```diff
@@ -194,11 +196,24 @@ def forms_info(self):
                 forms_info.append((module.case_type, form))
         return forms_info
 
+    @property
+    @memoized
+    def case_sharing_app_forms_info(self):
```

Comment:
Didn't realize `forms_info` was something that already existed...still don't think it's named well, but doesn't need to be dealt with in this PR.